test/browser_tests/page_objects/page_about-us.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function AboutUs() {
+  this.get = function() {
+    browser.get( '/about-us/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+  this.sidebar = element.all( by.css( 'aside.activity' ) );
```

Comment:
Should `sidebar` be named `activityBlock` or similar since it's not the sidebar but one of the activity feed blocks that gets loaded into the sidebar?